projects/plugins/backup/changelog/update-php-requirements

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
As above, this should probably be a major bump since it's dropping support for old PHP versions. OTOH, I'm not sure just how closely we care to follow semver in this respect, I'm sure we haven't been doing major bumps every time we dropped support for old WordPress versions (even though by a strict reading we probably should have been doing it for those too).

(Yeah, this doesn't actually matter for backup, jetpack, or videopress since they use wordpress versioning rather than semver, but we may as well set the significance right for all the plugins)

Reply:
👍 I switched to a major version bump in 6caa55c72efd1f4cd8352fb39b30055590d6efbc